packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v7.0.0.s.tool.mdx

```diff
@@ -0,0 +1,150 @@
+|`navConfig.navigation.primaryNavigationItems.capabilities`|**removed**|This configuration has been removed. All pages except for home will have access to devTools|
+|`pagesConfig`|**removed**|`pagesConfig` allowed consumers to provide their own page config. It was poorly documented, complex and no one ever did.|
+|`generatePages`|`additionalSearchDirectories`|Instead of filling out this complex object. Terra dev site searches the 'terra-dev-site' folder by default for content with the specified content Extensions. Consumers that wish to search other directories now can provided fully qualified paths which will also be searched|
+|`hotReloading`|**removed**|True in dev false in prod|
+|`npmPackage`|**removed**|Only the root npm package is used.|
+|`sideEffectImports`|`sideEffectImportFilePaths`|The name now includes 'filePaths' to better describe the expected input.|
+|`placeholderSrc`|**removed**|No longer used.|
+|`readMeContent`|`primaryNavigationItems.additionalContent`|No longer a one off, consumers can now add additional content to each primary navigation item that would not normally be found by searching.|
+|`appConfig.title`|`titleConfig.title`|Moved to a titleConfig object to align with terra-application-navigation config.|
+|`appConfig.favicon`|`faviconFilePath`|No longer nested under appConfig. The name now includes 'filePaths' to better describe the expected input.|
+|`appConfig.themes`|**removed**|The available themes are now pulled from the terra-theme.config file via the webpack global variable `TERRA_THEME_CONFIG`.|
+|`appConfig.defaultTheme`|`defaultTheme`|No longer nested under appConfig.|
+|`appConfig.locales`|**removed**|Deprecated as of terra-dev-site 6.23, now removed.|
+|`appConfig.defaultLocale`|`defaultLocale`|No longer nested under appConfig.|
+|`appConfig.defaultDirection`|`defaultDirection`|No longer nested under appConfig.|
+|`appConfig.extensions`|`extensionItems`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.iconPath`|`extensionItems.iconPath`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.key`|`extensionItems.key`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.text`|`extensionItems.text`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.componentPath`|`extensionItems.modalFilePath`| Renamed to prepare for application v2 modals.|
+|`appConfig.extensions.size`|**removed**|One size fits all.|
+|`appConfig.headHtml`|`headHtml`|No longer nested under appConfig.|
+|`includeTestEvidence`|**removed**|Removed the ability to include test evidence, this should be covered now by a separate site report.|
```

Comment:
Is this a terra-toolkit report that teams need to enable?

Reply:
It's automatically enabled for all 'orion' builds internally.